code/game/objects/items/scanners.dm

```diff
@@ -151,6 +151,7 @@ REAGENT SCANNER
 /obj/item/healthanalyzer/ui_static_data(mob/user)
 	var/list/data = list(
 		"patient" = patient.name,
+		"species" = patient.species.species_flags,
```

Comment:
Why is this here? It's incorrect and you just override it in the other line you added below.

Reply:
Thought I was defining it here and making it a list or something downstairs. Fixed.